src/services/s3/s3-utils.ts

```diff
@@ -1,11 +1,15 @@
 import { GetObjectCommand, S3 } from "@aws-sdk/client-s3";
-import Config from "../../config";
+import { Config, BucketName } from "../../config";
 import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
 import { createPresignedPost } from "@aws-sdk/s3-presigned-post";
 
-export async function postResumeUrl(userId: string, client: S3) {
+export async function postResumeUrl(
```

Comment:
nit: rename the get and post function names because now these functions will be getting URLs for both resumes and for speaker images

Reply:
so it's no longer just getResumeURL/postResumeURL